src/install/memcached/intro.mdx

```diff
@@ -0,0 +1,39 @@
+---
+headingText: Before we start
+componentType: default
+---
+
+import infrastructureMongoDashboardCropped from 'images/infrastructure_screenshot-crop_mongodb-dashboard-cropped.webp'
+
+Our Memcached [integration](/docs/integrations/host-integrations/getting-started/introduction-host-integrations) collects and sends inventory and metrics from your Memcached instance to our platform, where you can aggregate and visualize key performance metrics. We collect data at both instance and slab levels.
+
+
+To get the most out of this page, select the installation method that fits your environment. You need a [New Relic account](https://newrelic.com/signup) before starting the installation process.
```

Comment:
for some reason, this content isn't appearing in the deploy preview from netlify. am going to open it up in the local host but wanted to call that out

Reply:
fixed!